cmd/flowserver/main.go

```diff
@@ -33,6 +37,7 @@ import (
 )
 
 var version = "dev"
+var assetManager flows.AssetManager
```

Comment:
This asset manager will exist for the lifetime of the server and be shared by all requests